packages/ripple-keypairs/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Cryptographic key pairs for the XRP Ledger",
   "scripts": {
     "build": "tsc -b",
-    "test": "tsc -b && nyc mocha",
+    "test": "jest --verbose false --silent=false --runInBand ./test/*.test.ts",
```

Comment:
Does this need to have `--runInBand`? I assume not

Reply:
Changed